skgstat/Variogram.py

```diff
@@ -2111,6 +2111,103 @@ def to_gstools(self, **kwargs):
         """
         return skgstat_to_gstools(self, **kwargs)
 
+    def to_krige(self, **kwargs):
+        """
+        Instatiate a GSTools Krige class.
+
+        This can only export isotropic models.
+        Note: the `fit_variogram` is always set to `False`
+
+        Parameters
+        ----------
+        variogram : skgstat.Variogram
+            Scikit-GStat Variogram instamce
+        **kwargs
+            Keyword arguments forwarded to GSTools Krige.
```

Comment:
This could be a good point for a reference to the GSTools documentation with intersphinx:
https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html

So you could refrain from copying the whole doc from GSTools (which might could change, as you have already pointed out)

Reply:
Yeah, I'll give that a try as well.